### PR TITLE
Feature/14102 dacpack deploymnet

### PR DESCRIPTION
**Pull request by steves-bjss**
Update to allow data seeding and database population via pipeline.

Activity AB#14975
Task AB#15219

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -8,10 +8,10 @@ variables:
   - group: dev-acr-secrets
 
 trigger:
-- main
+- feature/14102-dacpack-deploymnet
 
 pr:
-- main
+- feature/14102-dacpack-deploymnet
```
Comment (PhysicsJon) [inline azure-pipelines.yml:14]: Do these need reverting before the merge to main?

File: database/NHSD.GPITBuyingCatalogue.Database/NHSD.GPITBuyingCatalogue.Database.sqlproj [hunk 2]
```diff
@@ -227,6 +232,9 @@
     <None Include="PostDeployment\InsertAssociatedServices.sql" />
   </ItemGroup>
   <ItemGroup>
-    <PostDeploy Include="PostDeployment\PostDeployment.sql" />
+    <None Include="PostDeployment\PostDeployment.sql" />
   </ItemGroup>
+  <PropertyGroup>
+    <PreBuildEvent>powershell.exe -ExecutionPolicy Bypass -NoProfile -NonInteractive -File $(ProjectDir)BuildUtils\ScriptMerge.ps1</PreBuildEvent>
+  </PropertyGroup>
 </Project>
```
Comment (PhysicsJon) [inline database/NHSD.GPITBuyingCatalogue.Database/NHSD.GPITBuyingCatalogue.Database.sqlproj:240]: EoF newline needed

**Review by PhysicsJon — CHANGES_REQUESTED** [review]
Fairly minor changes required

File: azure-pipelines.yml (hunk 1 above)
Comment (steves-bjss) [reply azure-pipelines.yml]: Good spot, missed pushing my last commit.

File: database/NHSD.GPITBuyingCatalogue.Database/NHSD.GPITBuyingCatalogue.Database.sqlproj (hunk 2 above)
Comment (steves-bjss) [reply database/NHSD.GPITBuyingCatalogue.Database/NHSD.GPITBuyingCatalogue.Database.sqlproj]: done

**Review by PhysicsJon — APPROVED** [review]

**Pull request merged**